powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/AssertionController.java

```diff
@@ -53,8 +54,13 @@ public AssertionOptionsResponse options(@Valid @RequestBody final AssertionOptio
     }
 
     @PostMapping
-    public AssertionVerificationResponse verify(@Valid @RequestBody final VerifyAssertionRequest request) throws PowerAuthClientException {
-        return assertionService.authenticate(request);
+    public AssertionVerificationResponse verify(@Valid @RequestBody final VerifyAssertionRequest request, final HttpSession session) throws PowerAuthClientException {
+        final AssertionVerificationResponse response = assertionService.authenticate(request);
+        if (response.isAssertionValid()) {
```

Comment:
May we log the result (success/error) or is it already done in the service?

Reply:
The JS has verbose logging, but I added some logging to the service layer too.